app/views/forms/businesses/hires/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_with model: @form, url: {controller: "forms/businesses/hires", action: "create"} do |form| %>
```

Comment:
I think we can do `form_with model: [:forms, :businesses, :hires, @form]` with the current name spacing.

Reply:
Was able to use `<%= form_with model: @form do |form| %>` here (in upcoming refactor).